docs/user/FAQ.md

```diff
@@ -1,5 +1,55 @@
 # FAQ
 
-## Q: On Mac OS X, I neither can delete a node template nor a relationship template.
+## Q: What is TOSCA?
+
+A: Topology and Orchestration Specification for Cloud Applications (TOSCA) is an OASIS specification, which enables the modeling, provisioning and management of cloud applications. 
+For more detail, please click [here](http://eclipse.github.io/winery/tosca/).
+
+## Q: What is a CSAR?
+
+A: Cloud Service Archive (CSAR) is a packaging format defined by the TOSCA specification, which enables to bundle modeled TOSCA components in a self-contained manner. 
+In winery, you can model a service template and export it as a CSAR. 
+This CSAR can be loaded into the OpenTOSCA container in order to deploy your application. 
+
+## Q: How can I start the OpenTOSCA ecosystem?
+
+A: You can start the ecosystem by simply using Docker Compose or by using installation scripts. 
+Please refer to the [OpenTOSCA Getting Started guide](http://www.opentosca.org/sites/use_opentosca.html) for more details.
+
+## Q: Is there an open repository for TOSCA types?
+
+A: Yes! We provide a [github repository](https://github.com/OpenTOSCA/tosca-definitions-public) compatible to Winery, which contains several service templates, node types, etc. 
+To use this repository with a Winery docker container, please refer to the corresponding [configuration instructions](https://github.com/OpenTOSCA/opentosca-docker#how-to-clone-a-private-tosca-definitions-repository-to-be-used-with-winery).
+
+## Q: Where can I find a quick start guide to model node types?
+
+A: You can find a Winery quick start guide about modeling node types [here](quickstart.md).  
```

Comment:
Please do not use `here`, but rather the file name. E.g. `... node types in our [Quickstart](quickstart.md).`